gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsInfoApiMonitor.java

```diff
@@ -50,18 +56,48 @@ public ClusterStats monitor(ProxyBackendConfiguration backend)
     }
 
     private boolean isReadyStatus(String baseUrl)
+    {
+        return isReadyStatus(baseUrl, retries);
+    }
```

Comment:
This method seems redundant. L52 could call `isReadyStatus(String baseUrl, int retriesRemaining)` instead.